.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+name: CI
+on:
+  push:
+    branches:
+      - '*'
+      - '!dev'
+      - '!master' # remove once main branch is dev
+jobs:
+  build-and-test:
+    name: Run Build and UI Tests
+    runs-on: macos-latest
```

Comment:
does this really need to be built on a Mac?
They have linux build agents too (mac agents are usually slower to queue)

Reply:
The emulator action suggests using macos https://github.com/ReactiveCircus/android-emulator-runner

`It is recommended to run this action on a macOS VM, e.g. macos-latest or macos-10.15 to take advantage of hardware accleration support provided by HAXM.`